frontend/src/HomePage/Item/Food/FoodPane.tsx

```diff
@@ -21,7 +21,7 @@ function FoodPane({
   return (
     <Box>
       <Heading>食品</Heading>
-      <Flex gap={4}>
+      <Flex padding="5px" overflowX="auto" gap={4}>
```

Comment:
```suggestion
      <Flex p={1} overflowX="auto" gap={4}>
```

Chakra UIでは[スペース幅がテーマレベルで決められている](https://chakra-ui.com/docs/styled-system/theme#spacing)ので，そちらの使用をお願いします
ただし，値が`5px`でないとダメな場合は変更しなくて大丈夫です